unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -32,6 +32,7 @@ public enum EditModeState
 
     [Header("Activation of Feature")]
     public bool activeFeature = false;
+    public bool byPassLandOwnershipCheck = false;
```

Comment:
`bypass` is a single word.

```suggestion
    public bool bypassLandOwnershipCheck = false;
```